### ISSUE TITLE
[BUG] Unreliable page routing for documentation website

### ISSUE THREAD

**Issue by mrvaghani** [AmruthPillai/Reactive-Resume#668]
**Describe the bug**
There is some funny routing going on for the documentation website which sometimes causes the page to be not found.

**To Reproduce**
1. Visit the homepage of the repo - https://github.com/AmruthPillai/Reactive-Resume
2. Scroll down to"Build from Source" section, click on [Source Code](https://docs.rxresu.me/source-code) hyperlink
3. Click on either "Docker" or "Local Build" 
4. See Page Not Found error

**Expected behavior**
The page should load.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Firefox 98.0 (64-bit)
 - Version: 98.0 (64-bit)

**Additional context**
I noticed that the URL when getting referred from GitHub's README is "https://docs.rxresu.me/source-code". However, if you just add "/" at the end of the URL ("https://docs.rxresu.me/source-code/"), then the "Docker" and "Local Build" hyperlinks are updated properly.

TL:DR; "https://docs.rxresu.me/source-code" and "https://docs.rxresu.me/source-code/" are rendering different HTML content.

**Comment by AmruthPillai** [comment]
Thank you for raising this issue and bringing this to light, this has been fixed now :)
